say:
  ros__parameters:
    SPEAKER_DEVICE_NAME: "default"
    SPEAKER_INPUT_CHANNELS: 32
    SPEAKER_OUT_CHANNELS: 32

    SPEAK_SERVICE: REPLACE
    speak_topic: "/speech/speak_now"
    speaking_topic: "saying"
    robot_audio_topic: "/robot_audio_output" # For AEC reference

    model: "af_heart"
    offline: True
    TTS_SERVER_IP: "127.0.0.1:50050"
